### PR TITLE
#402: Don't fail if captureStackTrace is not a function

### PR DESCRIPTION
**Pull request by joovel**

### CODE REVIEW

File: lib/JsonWebTokenError.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 var JsonWebTokenError = function (message, error) {
   Error.call(this, message);
-  Error.captureStackTrace(this, this.constructor);
+  if(Error.captureStackTrace) {
+    Error.captureStackTrace(this, this.constructor);
+  }
```
Comment (ziluvatar) [inline lib/JsonWebTokenError.js]: I'm curious, have you investigated why this happens in Chrome? (it uses the same v8 as node). Could you take a look? Maybe we can improve the Error creation without this `if`.

Comment (joovel) [reply lib/JsonWebTokenError.js]: From the issue:

> I've started testing my application outside of Chrome and started receiving the error

So my understanding is that this function is specific to V8, meaning it _works_ Chrome and node and _not_ FF and Safari.

Here are some options: 
* http://codegists.com/snippet/javascript/errorsjs_girvo_javascript for a slightly better in-code solution
* https://www.npmjs.com/package/error-polyfill for a heavier-weight browser-only polyfill

Comment (ziluvatar) [reply lib/JsonWebTokenError.js]: :( I misread it, it's simpler your approach. I'll approve, I'll wait for merging to see if we have more fixes before I release a patch version (I mean currently).

**Comment by ziluvatar** [comment]
Released on v8.1.0. 🎉  Thanks @joovel !!

**Pull request merged**